### DISCUSSION TITLE
Concatenate two networks diagrams (VIS+VIS)

### DISCUSSION THREAD

**Discussion by EnterpriseT** [alephdata/aleph#1515] [Ideas]
Is there a way to concatenate two networks diagrams?

Would be great to have a feature in networks's toolbar, like "Export diagrams" to get another diagram with another.

Any method: manually through command line or with some apps?
Can someone has ideas? Maybe I can make it PR.

**Comment by pudo** [comment]
Hey @EnterpriseT - can you have a look at https://github.com/alephdata/aleph/blob/main/SUPPORT.md and tell us more about how you're using Aleph, please? This system is being developed by a journalistic NGO. We build this for journalists, but we cannot justify to our donors to take feature requests from corporate or government users.

**Comment by EnterpriseT** [comment]
Hey @pudo! Thanks for mentioning that doc (SUPPORT) with a detailed description. I fully support a philosophy of open source and clearly understand the limitation of maintaining such projects. For sure, maintainers can (and some time must) ignore feature request irrelevant to major objectives.

Unfortunately, my usecase is not related to journalism (at least not yet): we are using Aleph as an internal tool to manage thousands of documents with creating network diagrams to show internal processes and make them much more clear. But I believe that some features/functions will be useful for journalists also.

On top of Aleph, we are going to build an automatic diagram generator. It depends on keywords, extracting from documents. When we will finalize it - hope it will be useful for Aleph as well.
